.travis.yml

```diff
@@ -17,22 +19,16 @@ stages:
 language: node_js
 node_js: '10'
 
-addons:
```

Comment:
I'm removing this for now as looking back at the build history Cypress has always been using the built in Electron browser, so this is just adding time to the build (and in the case of last week, failing the build). We should revisit this if we decide there _is_ value running in the latest stable Chrome, but we are getting value out of the default Electron browser (which is a recent-ish chromium version under the hood) so I think we can stick to that for now.